hudi-spark-datasource/hudi-spark/src/test/java/org/apache/hudi/functional/TestNewHoodieParquetFileFormat.java

```diff
@@ -114,25 +117,42 @@ protected void runIndividualComparison(String tableBasePath) {
   }
 
   protected void runIndividualComparison(String tableBasePath, String firstColumn, String... columns) {
-    Dataset<Row> legacyDf = sparkSession.read().format("hudi")
-        .option(HoodieReaderConfig.FILE_GROUP_READER_ENABLED.key(), "false")
-        .load(tableBasePath);
-    Dataset<Row> fileFormatDf = sparkSession.read().format("hudi")
-        .option(HoodieReaderConfig.FILE_GROUP_READER_ENABLED.key(), "true")
-        .load(tableBasePath);
-    if (firstColumn.isEmpty()) {
-      //df.except(df) does not work with map type cols
-      legacyDf = legacyDf.drop("city_to_state");
-      fileFormatDf = fileFormatDf.drop("city_to_state");
-    } else {
-      if (columns.length > 0) {
-        legacyDf = legacyDf.select(firstColumn, columns);
-        fileFormatDf = fileFormatDf.select(firstColumn, columns);
+    List<String> queryTypes = new ArrayList<>();
```

Comment:
Do we have tests cover RO queries on bootstrapped MOR table with updates in log files?  Only changing the validation may not be enough as we need to cover such cases particularly (might need to add preconditions to check that a MOR table has updates in log files for such cases).

Reply:
This test has updates in log files.